main.py

```diff
@@ -1137,7 +1162,9 @@ def load_interfaces_tags_values(switch: Switch,
             interface = switch.interfaces[port_number]
             interface.set_available_tags_tag_ranges(
                 available_tags,
-                interface_details['tag_ranges']
+                interface_details['tag_ranges'],
+                interface_details['special_available_tags'],
+                interface_details['special_tag_range'],
```

Comment:
The DB script needs to be updated to also set `special_tag_range` since this is assuming the key will be there (what I'm also suggesting to name as `special_tags` or anything else similar that you might suggest)

Reply:
Script updated